tests/topotests/lib/common_config.py

```diff
@@ -665,6 +667,9 @@ def generate_support_bundle():
         rnode.run("mkdir -p {}".format(dst_bundle))
         rnode.run("mv -f {}/* {}".format(src_bundle, dst_bundle))
 
+    """
+    pass
```

Comment:
What does `pass` do? Skips the function? Why is this code being commented with `"""`?